app/main.py

```diff
@@ -2,0 +1,12 @@
+def copy_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+
+    src_file, dst_file = parts[1], parts[2]
+    if src_file == dst_file:
+        return
+
+    with open(src_file, "r") as file_in, open(dst_file, "w") as file_out:
```

Comment:
The use of two context managers in a single `with` statement is correct and follows the example provided in the task description.